### PR TITLE
17706 Restarts overview page

### PR DESCRIPTION
**Pull request by GeorgeChatzigiannis**
UI built as per design

### CODE REVIEW

**Comment by jon-shipley** [comment]
@GeorgeChatzigiannis - can you add a better description?

File: admin/spec/controllers/restart.spec.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    beforeEach(() => {
+      sandbox = sinon.sandbox.create()
+    })
+
+    afterEach(() => {
+      sandbox.restore()
+    })
+
+    it('displays the restart overview page when redirected from home page', async (done) => {
```
Comment (jon-shipley) [inline admin/spec/controllers/restart.spec.js]: I don't see anything about redirection in the test body

File: admin/spec/controllers/restart.spec.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
```
Comment (jon-shipley) [inline admin/spec/controllers/restart.spec.js]: Again nothing about redirection in the test?

File: admin/views/restart/restart-overview.ejs [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        </header>
+        <br />
+        <span class="font-medium">
+            If a pupil was significantly interrupted while taking the check,
+            use this page to notify the STA that they will restart the check.
+            The pupil will take a different check form from their previous attempt
+            and the last completed attempt will be reported.
+        </span>
+    </div>
+    <div class="column-one-third add-title-margin" tabindex="0" aria-label="sidebar">
```
Comment (jon-shipley) [inline admin/views/restart/restart-overview.ejs]: Why put a tabindex=0 here?

File: admin/views/restart/restart-overview.ejs [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+        <aside role="complementary" class="support-column">
+            <h3 class="heading-medium" tabindex="0">Related</h3>
```
Comment (jon-shipley) [inline admin/views/restart/restart-overview.ejs]: Why add tabindex here?  GDS pattern is here: https://govuk-elements.herokuapp.com/typography/#typography-headings

**Comment by GeorgeChatzigiannis** [comment]
@jon-shipley improved description

File: admin/views/restart/restart-overview.ejs (hunk 4 above)
Comment (GeorgeChatzigiannis) [reply admin/views/restart/restart-overview.ejs]: @jon-shipley yes they aren't needed, removed

File: admin/views/restart/restart-overview.ejs (hunk 3 above)
Comment (GeorgeChatzigiannis) [reply admin/views/restart/restart-overview.ejs]: @jon-shipley as said removed

File: admin/spec/controllers/restart.spec.js (hunk 2 above)
Comment (GeorgeChatzigiannis) [reply admin/spec/controllers/restart.spec.js]: @jon-shipley the redirection text part is removed as it doesn't make sense

File: admin/spec/controllers/restart.spec.js (hunk 1 above)
Comment (GeorgeChatzigiannis) [reply admin/spec/controllers/restart.spec.js]: @jon-shipley fixed description

File: admin/spec/controllers/restart.spec.js [hunk 5]
```diff
@@ -0,0 +1,81 @@
+'use strict'
+/* global describe beforeEach afterEach it expect jasmine spyOn */
+const proxyquire = require('proxyquire')
+const httpMocks = require('node-mocks-http')
+const sinon = require('sinon')
```
Comment (jon-shipley) [inline admin/spec/controllers/restart.spec.js]: I wouldn't expect to see this in a controller.  Is it used?

Comment (GeorgeChatzigiannis) [reply admin/spec/controllers/restart.spec.js]: @jon-shipley sinon and proxyquire are not used.
Updated

**Pull request merged**